fmskill/comparison.py

```diff
@@ -977,15 +981,37 @@ def scatter(
             skill_df = self.skill(
                 metrics=["bias", "rmse", "urmse", "mae", "cc", "si", "r2"], df=df
             )  # df is filtered to matching subset
+            stats_with_units=["bias", "rmse", "urmse", "mae"]
+            
             lines = []
-
+            if units==None:
+                #Check for units
+                try:
+                    #Extract first letter of most common dfs0 units in SI;
```

Comment:
Formatting of units should be consistent, and it is better to put this in a more central place. 

There is already similar functionality (@jsmariegaard does it make sense to have this function in `observation.py`?)
https://github.com/DHI/fmskill/blob/3cb0e80e9536c03b71aa5e1f4e5b5290d13866d1/fmskill/observation.py#L488

Reply:
Thanks for the info I had not noticed. I rewrote the code in a very short line now by making use of the `unit_text` just a few lines above.